iib/workers/tasks/build.py

```diff
@@ -106,59 +106,70 @@ def _cleanup():
 
 
 @retry(exceptions=IIBError, tries=3, logger=log)
-def _create_and_push_manifest_list(request_id, arches):
+def _create_and_push_manifest_list(request_id, arches, build_tags):
```

Comment:
This will have to be refactored as a result of https://github.com/release-engineering/iib/pull/286